107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -228,10 +237,17 @@ private static void addIdentityCopierIfNoneRegistered(Map<Class<?>, ClassInstanc
     return new ExpiryPolicyToEhcacheExpiry<K, V>(config.getExpiryPolicyFactory().create());
   }
 
-  private <K, V> CacheLoaderWriter<K, V> initCacheLoaderWriter(CompleteConfiguration<K, V> config, MultiCacheException mce) {
+  private <K, V> JsrCacheLoaderWriter<K, V> initCacheLoaderWriter(CompleteConfiguration<K, V> config, MultiCacheException mce) {
     Factory<CacheLoader<K, V>> cacheLoaderFactory = config.getCacheLoaderFactory();
     Factory<CacheWriter<K, V>> cacheWriterFactory = getCacheWriterFactory(config);
```

Comment:
You strayed too close to this line... could we inline this method and annotate the declaration - rather than having a static method just for this?